lib/stackInformation.js

```diff
@@ -82,7 +82,9 @@ module.exports = {
 		.mapSeries(stack => BbPromise.join(BbPromise.resolve(stack), this.aliasStackLoadTemplate(stack)))
 		.map(stackInfo => ({ stack: stackInfo[0], template: stackInfo[1] }))
 		.catch(err => {
-			if (err.statusCode === 400) {
+			if (err.statusCode === 400
```

Comment:
What do you think about (to make it easier to understand):

```suggestion
if (
  err.statusCode === 400 ||
  (
    err.statusCode === undefined &&
    err.providerError &&
    err.providerError.statusCode === 400
  )
) {
```